packages/react/src/internal.tsx

```diff
@@ -11,6 +11,13 @@ export {
 } from './hooks/actions/useNavigateAction';
```

Comment:
Just curious, why is this file `.tsx`?

Reply:
Good point, should be a ts file. Will update.